MAINTAINERS.md

```diff
@@ -8,3 +8,4 @@
 |[lowzj](https://github.com/lowzj)|Jin Zhang|zj3142063@gmail.com| Alibaba Group|
 |[Starnop](https://github.com/Starnop)|Yuxing Liu|yuxing.lyx@alibaba-inc.com| Alibaba Group|
 |[zhouhaibing089](https://github.com/zhouhaibing089)|Haibing Zhou|zhouhaibing089@gmail.com| eBay |
+|[inoc603](https://github.com/inoc603)|Eddie Huang|inoc603@gmail.com| ByteDance |
```

Comment:
I am wondering if we could add the EddieHuang with the github ID in alphabetical order？@Starnop

Reply:
OK. I will update it.